### PR TITLE
feat: support `-L` and `-P` for `pwd`

### LINKED ISSUE #18: `pwd` only shows the physical path
Here I have `/home/luca` symlinked to `/home/luca/test`. `pwd` in `deno task`, not matter what argument I give it, will always show me the physical path (aka `pwd -P`). Currently there is no way to get the logical path with the `pwd` in `deno task`.
```console
$ deno task pwd
Warning deno task is unstable and may drastically change in the future
Task pwd pwd --help
/home/luca
$ pwd
/home/luca/test
$ pwd -P
/home/luca
```

### PR DESCRIPTION
**Pull request by dsherret**
Closes #18

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,6 +12,7 @@ description = "Cross platform scripting for deno task"
 [dependencies]
 anyhow = "1.0.55"
 futures = "0.3.16"
+path-dedot = "3.0.14"
```
Comment (dsherret) [inline Cargo.toml:15]: I tried path-clean, but it seemed to not be as good.

Comment (bartlomieju) [reply Cargo.toml:15]: Nice you found this one

File: src/commands/cd.rs [hunk 2]
```diff
@@ -33,7 +34,11 @@ pub fn cd_command(
 fn execute_cd(cwd: &Path, args: Vec<String>) -> Result<PathBuf> {
   let path = parse_args(args)?;
   let new_dir = cwd.join(&path);
-  let new_dir = fs_util::canonicalize_path(&new_dir)?;
+  let new_dir = match new_dir.parse_dot() {
```
Comment (bartlomieju) [inline src/commands/cd.rs:37]: This is very nice API 👍

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**